eos/b-decays/bq-to-dq-psd.cc

```diff
@@ -75,7 +75,9 @@ namespace eos
 
         UsedParameter mu;
 
-        RestrictedOption opt_accuracy;
+        UsedParameter DeltaP;
+
+        SpecifiedOption opt_accuracy;
```

Comment:
Why was this converted back to a SpecifiedOption? I think a RestrictedOption made more sense?